docker/Dockerfile

```diff
@@ -6,21 +6,23 @@ ARG APT_MIRROR="mirror://mirrors.ubuntu.com/mirrors.txt"
 # ARG APT_MIRROR="http://mirror.aarnet.edu.au/pub/ubuntu/archive/"
 
 # Configure mirror. Pass --build-arg APT_MIRROR=<mirror URL> to set a mirror if this is slow
-RUN sed -i "s#htt[p|ps]://archive.ubuntu.com/ubuntu/#$APT_MIRROR#g" /etc/apt/sources.list
+RUN sed -i "s#htt[p|ps]://archive.ubuntu.com/ubuntu/#$APT_MIRROR#g" /etc/apt/sources.list.d/ubuntu.sources
 
 # Install packages
 RUN \
     apt update && \
     apt install -y git python3 && \
-    apt install -y cmake gcc-arm-none-eabi libnewlib-arm-none-eabi build-essential
+    # For Pico SDK
+    apt install -y cmake gcc-arm-none-eabi libnewlib-arm-none-eabi libstdc++-arm-none-eabi-newlib build-essential
 
 # Install Pico SDK into a new stage
 FROM base as buildtools
 
+# Install Pico SDK
 RUN \
     mkdir -p /pico/ && \
     cd /pico/ && \
-    git clone https://github.com/raspberrypi/pico-sdk.git --branch 1.5.1 && \
+    git clone https://github.com/raspberrypi/pico-sdk.git --branch 2.0.0 && \
```

Comment:
I went ahead and had a pass at bumping this to 2.1.1 (current release). I got errors about not installing the correct version of picotool (which is normally done automatically). It would be nice to sort that out eventually, but we can probably pass for now.

Note: I suspect the fix might be explicitly installing picotool from source, instead of relying on the SDK to do it for us.